src/main/java/de/tum/in/www1/exerciseapp/web/rest/ParticipationResource.java

```diff
@@ -106,6 +107,26 @@ public ParticipationResource(ParticipationService participationService, Particip
     }
 
     /**
+     * POST  /courses/:courseId/exercises/:exerciseId/resume-participation: resume the participation of the current user in the exercise identified by id
+     *
+     * @param courseId   only included for API consistency, not actually used
+     * @param exerciseId id of the exercise for which to resume participation
+     * @param principal  current user principal
+     * @return ResponseEntity with status 200 (OK) and with updated participation as a body, or with status 500 (Internal Server Error)
+     */
+    @PutMapping(value = "/courses/{courseId}/exercises/{exerciseId}/resume-participation")
+    @PreAuthorize("hasAnyRole('USER', 'TA', 'ADMIN')")
+    @Timed
+    public ResponseEntity<Participation> resumeParticipation(@PathVariable Long courseId, @PathVariable Long exerciseId, Principal principal) throws URISyntaxException {
+        log.debug("REST request to resume Exercise : {}", exerciseId);
+        Exercise exercise = exerciseService.findOne(exerciseId);
```

Comment:
Please check that this is a programming exercise, otherwise log it and do nothing

Reply:
Done. I still return 200 though, I guess there is no reason to return 500.